lib/parallel_tests/cli.rb

```diff
@@ -182,6 +185,19 @@ def parse_options!(argv)
       options
     end
 
+    def parse_additional_test_options(test_opts, argv)
+      rspec_opts = argv.dup
```

Comment:
index() is iteration under the hood anyway, so I'm guessing you mean in terms of readability - but this gets complex if you consider the edge cases. The while loop is actually a clean way to iterate backwards, and given the actual use case here (maybe a dozen args or so, and it's parsing during startup - so we're not processing big data here) there's no performance to go after, which getting the indexes right is actually tricky and counterproductive -- as you can see here:  https://github.com/e2/parallel_tests/blob/fix_402_alternative_optional_arguments/lib/parallel_tests/cli.rb#L194-L207